src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -167,11 +163,17 @@ export class DictionaryConcepts extends Component {
 
   handleDelete = () => {
     const { data, collectionName, type } = this.state;
-    this.props
-      .removeDictionaryConcept(data, type, this.props.match.params.typeName, collectionName);
-  }
+    this.props.removeDictionaryConcept(
+      data,
+      type,
+      this.props.match.params.typeName,
+      collectionName,
+    );
+  };
 
-  handleShowDelete = (url) => { this.setState({ data: { references: [url] }, versionUrl: url }); };
+  handleShowDelete = (url) => {
+    this.setState({ data: { references: [url] }, versionUrl: url });
```

Comment:
I have noticed quite a number of formatting changes, don't you think that would be best handled by a ticket of of its own?

Reply:
@alexmochu  I have reverted all the formatting changes.

Reply:
@Michyjones No you have not, I can still see the changes.

Reply:
Can you please share the ticket that will handle the linting errors above.

Reply:
@alexmochu I realised that I had configured my eslint wrongly. There are no more highlighting errors.